src/app/body/create-new-session/create-new-session.component.ts

```diff
@@ -29,7 +30,7 @@ export class CreateNewSessionComponent implements OnInit {
   todayDate: string
   description: string
   assigneeName: string
-  creatorName: string
+  creatorName = this.authService.user.email
```

Comment:
Add the specification and assign an empty element. And use authService in ngOnInit()

Reply:
![creatorName](https://user-images.githubusercontent.com/74500169/129551242-d7d310ba-7ee4-4f1c-b023-b0754db92d16.png)

Now, are these changes enough?